PractiseMay11/PlantsTask/Gallery.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseMay11.PlantsTask
+{
+    internal class Gallery
+    {
+        public Plant[] plants;
```

Comment:
plants and workers should be something hidden for managing outside the gallery. It is a kind of incapsulation.